pkg/policies/branch/branch.go

```diff
@@ -267,6 +288,42 @@ func check(ctx context.Context, rep repositories, c *github.Client, owner,
 				d.BlockForce = false
 			}
 		}
+		rsc := p.GetRequiredStatusChecks()
+		if rsc != nil {
+			d.RequireUpToDateBranch = rsc.Strict
+			if mc.RequireUpToDateBranch && !rsc.Strict {
```

Comment:
Would update this to `if mc.RequireUpToDateBranch && !rsc.Strict && len(mc.RequireStatusChecks) > 0`

Reply:
Updated in 754144bd3f9e24f3554793450990d44d4116b98f by having `len(mc.RequireStatusChecks) > 0` at the top level.